packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1066,6 +1100,13 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         }
 
         ReportOpPerfTelemetry(this.context.clientId, this.deltaManager, this.logger);
+
+        this.logger.sendTelemetryEvent({
```

Comment:
It would be great if we could reduce number of events overall, and use existing "ContainerLoadStats" event somehow. "AttachmentBlobsLoaded" is another event like that.
They are all in different class, but maybe you can see some natural way for info from child classes to be returned to ContainerRuntime who logs all of this error together in one telemetry statement? Like maybe returning data (properties) from function calls?

Reply:
using ContainerLoadStats should work. ContainerRuntime actually already pass metadata to DataStores indirectly